pkg/driver/helmdriver.go

```diff
@@ -35,6 +35,10 @@ type helmDriver struct {
 
 var _ PackageDriver = (*helmDriver)(nil)
 
+// TODO: Temporarily hard coding but needs to be read from secrets
+var caFile = "/tmp/config/registry/CACERTCONTENT"
+var insecure = false
```

Comment:
insecure flag is in the registrymirrorsecret.yaml, you can add that as an env variable to deployment.yaml and then read from there